src/table/table.scss

```diff
@@ -130,7 +130,7 @@
         }
         user-select: none;
 
-        > .iui-icon {
+        .iui-icon {
```

Comment:
What this was needed for?

Reply:
Check PR description

Reply:
But this will be removed with the table refactor, because...
![image](https://user-images.githubusercontent.com/81580355/116974340-abc13480-acc6-11eb-8ace-a4b36cf07ec8.png)
??

Reply:
Then we can change the styling when that happens. I think this is fine for now idk